### PR TITLE
Refactoring routing table.

### PR DESCRIPTION
**Pull request by tzzed**
This PR of code refactoring. I am going to add some comments about some lines of code.

### CODE REVIEW

File: src/routing_table.c [hunk 1]
```diff
@@ -293,94 +247,82 @@ uint16_t RoutingTB_GetNodeID(uint16_t index)
...
  * @param nb to add
  * @return None
  ******************************************************************************/
-static void RoutingTB_AddNumToAlias(char *alias, uint8_t num)
-{
+static void RoutingTB_AddNumToAlias(char *alias, uint8_t num) {
     uint8_t intsize = 1;
-    if (num > 9)
-    {
+    if (num > 9) {
```
Comment (tzzed) [inline src/routing_table.c:272]: What is `9` a comment or a macro would be useful.

File: src/routing_table.c [hunk 2]
```diff
@@ -293,94 +247,82 @@ uint16_t RoutingTB_GetNodeID(uint16_t index)
...
         // The string size of num is 2
         intsize = 2;
     }
-    if (num > 99)
-    {
+    if (num > 99) {
```
Comment (tzzed) [inline src/routing_table.c:276]: Same as above.

File: src/routing_table.c [hunk 3]
```diff
@@ -293,94 +247,82 @@ uint16_t RoutingTB_GetNodeID(uint16_t index)
...
         // This is probably a mistake, put an error into the alias
         memset(alias, 0, 15);
         memcpy(alias, "error", strlen("error"));
         return;
     }
     // Change size to fit into 15 characters
-    if (strlen(alias) > (15 - intsize))
-    {
+    if (strlen(alias) > (15 - intsize)) {
```
Comment (tzzed) [inline src/routing_table.c:283]: Same here. Why `15`.

File: src/routing_table.c (hunk 1 above)
Comment (Salem-Luos) [reply src/routing_table.c:272]: above 9 -> 2 decimal. seem obvious

File: src/routing_table.c (hunk 2 above)
Comment (Salem-Luos) [reply src/routing_table.c:276]: I will add a comment not a macro to explain the 99 limitation

File: src/routing_table.c (hunk 3 above)
Comment (Salem-Luos) [reply src/routing_table.c:283]: a macro will be add in the release

**Review by Salem-Luos — COMMENTED** [review]
your code refactoring not fit with Luos coding rules. we use coding template that seem more clear when you read the code

File: src/routing_table.c (hunk 1 above)
Comment (tzzed) [reply src/routing_table.c:272]: You check here the length of decimal in `num`  this is not the first thing come in mind. Seem obvious for you, but for us as a contributor is not. I think to be more explicit is better.

**Comment by nicolas-rabault** [comment]
Those subjects have been managed in release 1.3 and before.
Thanks @tzzed

**Pull request closed without merging**